validate_maintainers.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+"""
+Validate maintainer JSON files against the schema.
+
+Usage:
+    python validate_maintainers.py [--fix] [file.json ...]
+    
+Options:
+    --fix    Auto-fix common issues (case sensitivity, known aliases)
+    
+If no files are provided, validates all JSON files in content/maintainers/
+"""
+
+import json
+import sys
+from pathlib import Path
+
+try:
+    import jsonschema
+except ImportError:
+    print("Error: jsonschema not installed. Run: pip install jsonschema")
+    sys.exit(1)
```

Comment:
```suggestion
except ImportError as e:
    sys.exit(f"Error: jsonschema not installed. Run: pip install jsonschema. Error: {e}")
```

`sys.exit` has a pattern where if the argument is a string, it'll exit with a return code of 1 automatically ;) see https://docs.python.org/3/library/sys.html#sys.exit